main.cpp

```diff
@@ -260,19 +259,21 @@ int Tree::grow(int sunLight, int water)
     return sunLight * water;
 }
 // I couldn't quite figure out how to make this return a Fruit 
-// Tree::Fruit produceFruit(int grow)
-// {
-//      if (grow > 0)
-//      {
-//          return
-//             bool readyToEat = true;
-//      }
-//     else 
-//      {      
-//         return
-//             bool readyToEat = false;
-//     }
-// }
+Tree::Fruit produceFruit(int grow)
+{
+     if (grow > 0)
+     {
+        Tree::Fruit fruit;
+        fruit.readyToEat = true;
+        return fruit;
+     }
+    else 
+     {      
+        Tree::Fruit fruit;
+        fruit.readyToEat = false;
+        return fruit;
+    }
+}
```

Comment:
It's time to learn about Don't Repeat Yourself. (D.R.Y.)
There are a lot of repeated lines in this chunk of code. You need to revise it and remove the repetition.
- [ ] revise

Reply:
Thanks for that insight